### DISCUSSION TITLE
Add an option to disable building on nether roof.

### DISCUSSION THREAD

**Discussion by RealTriassic** [PurpurMC/Purpur#518] [__Dismissed__] [answered]
I'd think this option would be useful for servers that don't want players building on top of the nether roof, whilst still allowing players to walk on the roof. Yes, I am aware that Paper has an option for nether ceiling damage, but I still want players to be able to walk on the roof If they wish to do so.

Another small advantage that probably doesn't really matter, but I thought I'd still add it in:

- This option would also be useful for Geyser users as Bedrock Edition currently cannot render blocks on the nether ceiling, but they can walk on the nether ceiling without any problems otherwise.

**Comment by encode42** [comment]
This is better off in a plugin, and a nice starter one!

On block place → if in nether → if above roof (y level) → cancel event

**Comment by RhythmicSys** [comment, marked as answer]
Has been added to [PurpurExtras](https://github.com/PurpurMC/PurpurExtras/wiki#block-building-above-nether) now
